cyclonedx/output/json.py

```diff
@@ -56,28 +56,44 @@ def generate(self, force_regeneration: bool = False) -> None:
         if self.generated and not force_regeneration:
             return
 
+        self.get_bom().validate()
+
         schema_uri: Optional[str] = self._get_schema_uri()
         if not schema_uri:
             raise FormatNotSupportedException(
-                f'JSON is not supported by CycloneDX in schema version {self.schema_version.to_version()}'
-            )
-
-        vulnerabilities: Dict[str, List[Dict[Any, Any]]] = {"vulnerabilities": []}
-        if self.get_bom().components:
-            for component in cast(List[Component], self.get_bom().components):
-                for vulnerability in component.get_vulnerabilities():
-                    vulnerabilities['vulnerabilities'].append(
-                        json.loads(json.dumps(vulnerability, cls=CycloneDxJSONEncoder))
-                    )
+                f'JSON is not supported by CycloneDX in schema version {self.schema_version.to_version()}')
+
+        extras = {}
+        if self.bom_supports_dependencies():
+            dependencies: List[Dict[str, Union[str, List[str]]]] = []
+            if self.get_bom().metadata.component:
+                dependencies.append({
+                    'ref': str(cast(Component, self.get_bom().metadata.component).bom_ref),
+                    'dependsOn': [*map(str, cast(Component, self.get_bom().metadata.component).dependencies)]
+                })
+            if self.get_bom().components:
```

Comment:
no need for the condition. as far as i see `self.get_bom().components` is always a set.